assets/lokomotive-kubernetes/bootkube/resources/charts/lokomotive.yaml

```diff
@@ -0,0 +1,3 @@
+certificates:
+  caKey: ${ca_key}
```

Comment:
why caKey? Shouldn't we just pass serving cert and serving key to the deployment?

Reply:
Needed to create secret

Reply:
Yes, but this is not a CA certificate/key which you pass, but serving certificate as far as I understand. Terraform generates the certificate.

Reply:
Done